### DISCUSSION TITLE
I can't use unique and nullable at the same time

### DISCUSSION THREAD

**Discussion by Skipperlla** [prisma/prisma#15095] [Q&A] [answered]
Hello, in my application, I prepared it based on getting 1 data on each page, but if someone has already created a data, it gives such an error because the userName or email in the new data to be added is null.

![This my schema](https://cdn.discordapp.com/attachments/734689773070843995/1014255468841029733/Ekran_Resmi_2022-08-30_22.27.52.png)

![This is error](https://cdn.discordapp.com/attachments/734689773070843995/1014255469327552523/Ekran_Resmi_2022-08-30_22.28.11.png)

I used the mongodb

**Comment by austincrim** [comment, marked as answer]
Hey @Skipperlla 👋,

MongoDB unique indexes only [allow a single `NULL` value] (https://www.mongodb.com/docs/v5.0/core/index-unique/#unique-index-and-missing-field)across the entire collection.

If you have existing data that is null, you will need to add unique values before proceeding.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
